.github/workflows/auto-pr-rebuild-script.yml

```diff
@@ -70,11 +70,16 @@ jobs:
         run: |
           chmod +x ./rebuild.sh
           ./rebuild.sh
-      
-      - name: Discard changes in nargo directory
-        run: git restore --source=HEAD --staged --worktree -- nargo/
-    
+
+      - name: Check for changes in acir_artifacts directory
+        id: check_changes
+        run: |
+          git diff --quiet tooling/nargo_cli/tests/acir_artifacts/ || echo "::set-output name=changes::true"
+        outputs:
+          changes: ${{ steps.check_changes.outputs.changes }}
```

Comment:
Not sure this is necessary.

Reply:
Ah thought you needed to define the outputs, I removed it (I cannot test with ACT for some reason)